pom.xml

```diff
@@ -81,6 +81,13 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <maven.compiler.source>11</maven.compiler.source>
       <maven.compiler.target>11</maven.compiler.target>
+
+      <!-- plugins version -->
+      <version.maven.gpg>3.0.1</version.maven.gpg>
+
+      <!-- maven repository urls -->
+      <maven.releases.nexus.url>https://s01.oss.sonatype.org/</maven.releases.nexus.url>
+      <maven.releases.repo.id>ossrh</maven.releases.repo.id>
```

Comment:
Can we avoid the need to define this and the various plugins by simply inheriting from `infinispan-build-configuration-parent` instead of `infinispan-parent`?

Reply:
Updated. I've had to trim some bits to work properly.